torch/testing/_internal/logging_utils.py

```diff
@@ -0,0 +1,77 @@
+import torch._dynamo.test_case
+import unittest.mock
+import os
+import contextlib
+
+import logging
+
+def log_settings(settings):
+    return unittest.mock.patch.dict(os.environ, {"TORCH_LOGS": settings})
+
+def make_test(settings, log_names, install_hook=None):
```

Comment:
Unlike the other make_test in the codebase, this is in a utility model. As you are importing this unqualified, I think you will have more clarity if you say `make_logging_test`.